pancancer_evaluation/utilities/analysis_utilities.py

```diff
@@ -94,12 +94,19 @@ def load_prediction_results_lasso_range(results_dir,
         gene_dir = os.path.join(results_dir, gene_name)
         if not os.path.isdir(gene_dir): continue
         for results_file in os.listdir(gene_dir):
-            if not ('classify_metrics' in results_file): continue
+            if not ('classify' in results_file): continue
```

Comment:
Because of the changes I made to how I'm doing file naming in `file_utilities.py`, 'classify' and 'utilities' aren't necessarily always together in the file names. Separating them makes the code work for both the old file naming scheme and the new, more flexible one.